.github/workflows/r-lib-check.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: R-CMD-check (r-lib)
```

Comment:
Should we keep this as `R-CMD-check`? There now are two actions in the actions tab and it is not obvious which is the most current R-CMD-check without prior knowledge that this one is more recent.

![image](https://user-images.githubusercontent.com/1639487/210464957-31aa5162-fb20-4622-8a67-ba1063f2aec1.png)

Reply:
Good point. Originally, I differentiated so that we could compare the two actions. Now that the r-lib version has replaced the Docker version, perhaps I should rename it to just `R-CMD-check` now.